### PR TITLE
Business - Added Bank model and view

### PR DESCRIPTION
**Pull request by barys21**

### CODE REVIEW

**Review by DanyeIIo — APPROVED** [review]
solve conflicts and merge

File: UtopiaCity/Controllers/Business/BusinessController.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace UtopiaCity.Controllers.Business
+{
+    public class BusinessController: Controller
+    {
+        public IActionResult Index()
+        {
+            return View();
```
Comment (crocomoth) [inline UtopiaCity/Controllers/Business/BusinessController.cs:13]: is this needed?

**Review by crocomoth — APPROVED** [review]

**Comment by barys21** [comment]
solve conflicts and merge

File: UtopiaCity/Controllers/Business/BusinessController.cs (hunk 1 above)
Comment (barys21) [reply UtopiaCity/Controllers/Business/BusinessController.cs:13]: yes this is main business page

**Pull request merged**